fix(docs): update info about sitemap (#23074)
expo/expo: docs/pages/router/reference/sitemap.mdx

  style={{ maxWidth: 720 }}
/>

> Expo Router currently injects a **/\_sitemap** which provides a list of all routes in the app. This is especially useful for debugging.

Reviewer: ```suggestion
Expo Router currently injects a **/\_sitemap** automatically that provides a list of all routes in the app. This is useful for debugging.
```

> This can be removed by adding a **app/\_sitemap.js** file and returning a null component:

Reviewer: ```suggestion
To remove it, add a **app/\_sitemap.js** and return a null component:
```
